syncing/src/main/kotlin/maru/syncing/ELSyncService.kt

```diff
@@ -61,41 +63,37 @@ class ELSyncService(
   private val log = LogManager.getLogger(this.javaClass)
 
   private var poller: Timer? = null
-  private var lastELSyncTarget: ElBlockInfo? = null
   private var currentELSyncStatus: ELSyncStatus? = null
 
   private fun pollTask() {
     val latestBeaconBlockHeader = beaconChain.getLatestBeaconState().latestBeaconBlockHeader
     val latestBeaconBlockNumber = latestBeaconBlockHeader.number
     if (latestBeaconBlockNumber == 0UL) {
       val newELSyncStatus = ELSyncStatus.SYNCED
-      if (currentELSyncStatus != newELSyncStatus) {
-        currentELSyncStatus = newELSyncStatus
-        onStatusChange(newELSyncStatus)
-      }
+      currentELSyncStatus = newELSyncStatus
+      onStatusChange(newELSyncStatus)
       return
     }
 
     val latestSealedBeaconBlock =
       beaconChain.getSealedBeaconBlock(
         beaconBlockNumber = latestBeaconBlockHeader.number,
       )!!
+    val latestBeaconBlockBody = latestSealedBeaconBlock.beaconBlock.beaconBlockBody
     val newELSyncTarget =
       ElBlockInfo(
-        blockNumber = latestSealedBeaconBlock.beaconBlock.beaconBlockBody.executionPayload.blockNumber,
-        blockHash = latestSealedBeaconBlock.beaconBlock.beaconBlockBody.executionPayload.blockHash,
+        blockNumber = latestBeaconBlockBody.executionPayload.blockNumber,
+        blockHash = latestBeaconBlockBody.executionPayload.blockHash,
       )
-    if (lastELSyncTarget != newELSyncTarget) {
-      lastELSyncTarget = newELSyncTarget
-      log.info("New EL sync target = {}", newELSyncTarget)
-    }
+    log.info("New EL sync target = {}", newELSyncTarget)
```

Comment:
How often will this log be printed? It might be better at debug level if it's a lot or use the log throttle